engines/dradis-api/app/controllers/dradis/ce/api/v1/comments_controller.rb

```diff
@@ -11,16 +11,36 @@ def index
       def show; end
 
       def create
+        @comment = @commentable.comments.build(comment_params)
+        @comment.user = current_user
+        if @comment.save
+          track_created(@comment)
+          render status: 201, location: @commentable
```

Comment:
would this work? isn't this part of the reason why you created the custom weird polymorphic helper method